src/main/java/com/epam/webapphello/service/KneesUserService.java

```diff
@@ -0,0 +1,27 @@
+package com.epam.webapphello.service;
+
+import com.epam.webapphello.dao.UserDao;
+import com.epam.webapphello.entity.User;
+import com.epam.webapphello.exception.DAOException;
+import com.epam.webapphello.exception.ServiceException;
+
+
+import java.util.Optional;
+
+public class KneesUserService implements UserService{
```

Comment:
looks like a regular UserService 
Better name SimpleUserService is you want to tell that it's a temporary one

Reply:
ok

Reply:
remove